wallet/src/de/schildbach/wallet/service/NetworkState.kt

```diff
@@ -66,4 +67,8 @@ class NetworkState @Inject constructor(connectivityManager: ConnectivityManager)
             )
         }
     }
+
+    override fun isWifiConnected(): Boolean {
+        return connectivityManager.getNetworkInfo(ConnectivityManager.TYPE_WIFI)?.state == NetworkInfo.State.CONNECTED
+    }
```

Comment:
This came from another PR.  I thought this would be better than adding another StateFlow like **isConnected**.